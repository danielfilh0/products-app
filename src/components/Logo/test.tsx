import { render, screen } from '@testing-library/react'

import { Logo } from '.'

describe('<Logo />', () => {
  it('should render', () => {
    const { container } = render(<Logo />)

    expect(
      screen.getByRole('img', { name: 'Products App Logo' })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })

  it('should render by passing classes names', () => {
    const { container } = render(<Logo className="h-10" />)

    expect(container.getElementsByClassName('h-10').length).toBe(1)
  })
})
