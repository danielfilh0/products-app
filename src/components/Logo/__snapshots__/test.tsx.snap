// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should render 1`] = `
<img
  alt="Products App Logo"
  class="w-auto h-8"
  data-nimg="1"
  decoding="async"
  height="40"
  loading="lazy"
  src="/_next/image?url=https%3A%2F%2Ftailwindui.com%2Fimg%2Flogos%2Fmark.svg%3Fcolor%3Dindigo%26shade%3D600&w=96&q=75"
  srcset="/_next/image?url=https%3A%2F%2Ftailwindui.com%2Fimg%2Flogos%2Fmark.svg%3Fcolor%3Dindigo%26shade%3D600&w=48&q=75 1x, /_next/image?url=https%3A%2F%2Ftailwindui.com%2Fimg%2Flogos%2Fmark.svg%3Fcolor%3Dindigo%26shade%3D600&w=96&q=75 2x"
  style="color: transparent;"
  width="47"
/>
`;
